tools/dumppdf.py

```diff
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
```

Comment:
Let's use python3 as suggested by pep394 (thanks for the reference :) )

Reply:
Yes, done !